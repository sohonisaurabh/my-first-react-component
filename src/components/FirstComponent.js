import React from "react";

const FirstComponent = () => {
    return (
        <h2>Welcome to TDD using React!</h2>
    );
};

export default FirstComponent;