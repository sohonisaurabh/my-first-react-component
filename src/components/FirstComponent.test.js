import React from "react";
import { shallow } from "enzyme";
import FirstComponent from "./FirstComponent";

describe("FirstComponentBasicCheck", () => {
    it("Should render the component", () => {
        const wrapper = shallow(<FirstComponent />);
        expect(wrapper.getElements()).toMatchSnapshot();
    });
});
