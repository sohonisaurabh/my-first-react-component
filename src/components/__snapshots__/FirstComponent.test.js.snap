// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FirstComponentBasicCheck Should render the component 1`] = `
Array [
  <h2>
    Welcome to TDD using React!
  </h2>,
]
`;
